site/src/sphinx/server-cors-service.rst

```diff
@@ -0,0 +1,107 @@
+.. _CORSWiki: https://en.wikipedia.org/wiki/Cross-origin_resource_sharing
+.. _server-cors-service:
+
+Configure CORS service
+========================
+
+Armeria provides a way to configure Cross-origin resource sharing(CORS) policy for specified origins or
+any origin via :api:`CorsServiceBuilder`. For more information about CORS, visit CORSWiki_.
```

Comment:
How about:

   Visit the `Wikipedia's CORS <https://en.wikipedia...>`_ page for more information.